appcues/src/test/java/com/appcues/analytics/AutoPropertyDecoratorTest.kt

```diff
@@ -165,21 +165,13 @@ class AutoPropertyDecoratorTest {
         val event = EventRequest(
             name = AnalyticsEvent.SessionStarted.eventName,
         )
-        val sessionSets = mutableSetOf<Int>()
         // when
-        for (i in 0..10) {
-            // doing this ten times to make sure we account for some luck
-            with(autoPropertyDecorator.decorateTrack(event)) {
-                // then
-                with(attributes["_identity"] as Map<*, *>) {
-                    println("i is $i: ${get("_sessionRandomizer") as Int}")
-                    sessionSets.add(get("_sessionRandomizer") as Int)
-                }
+        with(autoPropertyDecorator.decorateTrack(event)) {
+            // then
+            with(attributes["_identity"] as Map<*, *>) {
+                assertThat(get("_sessionRandomizer") as Int).isEqualTo(10)
```

Comment:
im not sure if this is really testing the reset during SessionStarted is it? could we verify its 10, then update the mock to return a different value, send another session start, then verify the new value?